js/packages/cli/src/commands/updateFromCache.ts

```diff
@@ -62,6 +62,15 @@ export async function updateMetadataFromCache(
   cacheContent: any,
   newCacheContent: any,
 ) {
+  const utf8Encode = new TextEncoder();
+  const temp = await PublicKey.findProgramAddress(
+    [
+      utf8Encode.encode('candy_machine'),
+      new PublicKey(candyMachineAddress).toBytes(),
+    ],
+    new PublicKey('cndy3Z4yapfJBmL3ShUp5exZKqR3z33thTzeNMm2gRZ'),
```

Comment:
if you could update 71 and 68 to use the constants for these values, this will ensure smoother changes in the future for any updates.